gnat/vss_config.gpr

```diff
@@ -25,7 +25,7 @@
 abstract project VSS_Config is
 
    type Build_Mode_Type is ("dev", "prod", "coverage");
-   Build_Mode : Build_Mode_Type := external ("BUILD_MODE", "dev");
+   Build_Mode : Build_Mode_Type := external ("VSS_BUILD_MODE", "dev");
```

Comment:
It's better to have a default to production mode. User's of the library will most likely want best performances out of the box.

Reply:
I've checked with other IDE repos; "dev"/"debug" mode is default value, production environment selects "prod" mode during the builds.

Reply:
This is a bad default because people using this library will experience the worst performances.